tsl/src/compression/wal_utils.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+#include "guc.h"
+#include <replication/message.h>
```

Comment:
```suggestion
#pragma once

#include <postgres.h>
#include <replication/message.h>

#include "guc.h"
```
All headers should have the `#pragma once` to make sure it will be included only once. Also all headers should always include "postgres.h" as the first header.

Reply:
done